WalletWasabi/WabiSabi/Backend/Models/WabiSabiProtocolErrorCode.cs

```diff
@@ -37,6 +37,7 @@ public enum WabiSabiProtocolErrorCode
 		VsizeQuotaExceeded,
 		DeltaNotZero,
 		WrongNumberOfCreds,
-		CryptoException
+		CryptoException,
+		AlreadySigned
```

Comment:
This is `AlreadySignaled`, `ReadinessAlreadySignaled` or something like that.

Reply:
ACK